interface-definitions/policy-local-route.xml.in

```diff
@@ -177,6 +185,14 @@
                   <multi/>
                 </properties>
               </leafNode>
+              <leafNode name="incoming-interface">
+                <properties>
```

Comment:
Using the same XML definition two times is discouraged. Please create a .xml.i file and use `#include`